romanesco/executors/pyspark.py

```diff
@@ -0,0 +1,35 @@
+import imp
+import json
+import sys
+from romanesco import spark
+
+def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
+    custom = imp.new_module("custom")
+
+    # TODO
+    # Ensure that the spark modules have been added to our path
+    # Create SparkContext - Should we share one across a workflow?
+    # Take any inputs and convert them to RDDS?
+    print 'Running pyspark'
+
+    sc = spark.create_spark_context()
+
+    for name in inputs:
+        custom.__dict__[name] = inputs[name]["script_data"]
+        custom.__dict__['sc'] = sc
```

Comment:
That's a good point. We should certainly pull it out of the loop body :) I suggest:

```
sc = spark.create_spark_context()
custom.__dict__['sc'] = sc
custom.__dict__['spark_context'] = sc  # alias in case user-defined input is named "sc"

for name in inputs:
    custom.__dict__[name] = inputs[name]["script_data"]